force/sobjects.go

```diff
@@ -109,8 +109,12 @@ func (forceApi *ForceApi) DeleteSObject(id string, in SObject) (err error) {
 }
 
 func (forceApi *ForceApi) GetSObjectByExternalId(id string, fields []string, out SObject) (err error) {
+	return forceApi.GetSObjectBySpecificExternalId(id, fields, out.ExternalIdAPIName(), out)
+}
+
+func (forceApi *ForceApi) GetSObjectBySpecificExternalId(id string, fields []string, specificExternalId string, out SObject) (err error) {
```

Comment:
Should be `specificExternalIdAPIName`, or `specificExternalIdType`.

`specificExternalId` <- is confusing

Reply:
Also, tests?

Reply:
We wanted to test, but go-force tests actually talk to nimajalali's Salesforce account, and it would need to be set up to support our use case. We decided to punt.

This is maybe an indicator that go-force isn't a strong long-term solution for us.